### PR TITLE
implement latest_listing template

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@smn @miltontony @bruskiza ready for review

File: tuneme/templates/core/tags/latest_listing_homepage.html [hunk 1]
```diff
@@ -2,19 +2,31 @@
...
       <a href="{% pageurl page %}">{% image page.image width-60 %}</a>
       {% endif %}
-      <h3><a href="{% pageurl page %}">{{page.title}}</a></h3>
+      <h3>{{page.title}}</h3>
       <p>{{page.subtitle}}</p>
-    </li>
-  </ul>
+      <div class="comments-count"><a href=""><span>comments</span></a></div>
+      <a href="{% pageurl page %}" class="button">Read on</a>
+      <a href="{% pageurl page.get_parent_section %}" class="button inverted">View Similar</a>
```
Comment (smn) [inline tuneme/templates/core/tags/latest_listing_homepage.html]: Looking at the [template](http://praekelt.github.io/molo-tuneme/article.html) these links shouldn't be here I think?

Comment (smn) [inline tuneme/templates/core/tags/latest_listing_homepage.html]: _nevermind_ completely wrong template.

**Comment by smn** [comment]
:+1:

**Pull request merged**